addons/l10n_ar/views/uom_uom_view.xml

```diff
@@ -17,21 +17,9 @@
         <field name="model">uom.uom</field>
         <field name="inherit_id" ref="uom.product_uom_form_view"/>
         <field name="arch" type="xml">
-            <field name="rounding" position="after">
```

Comment:
these need to be double checked for layout issues
![image](https://github.com/user-attachments/assets/5210d982-4523-4827-85d6-36fc6ab10d73)

for reference, current layout:
![image](https://github.com/user-attachments/assets/fb70e0ea-edae-480a-8a0e-5e3a5fac494f)

Also, not sure why they made the `l10n_ar_afip_code` always visible in the list view, but only visible when using an ar company in the form view :shrug: